packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx

```diff
@@ -71,6 +73,44 @@ const ProposalForm = (): JSX.Element => {
 
         return { ...commonOptions, type: ProposalType.RemoveCommitteeMember, committee, address };
       }
+      case ProposalType.AmendElectionRuleThreshold: {
+        const targetElectionRuleId = parseInt(
+          values[COMMITTEE_THRESHOLD_FIELD].substring(0, values[COMMITTEE_THRESHOLD_FIELD].indexOf(":")),
+          10,
+        );
+        const thresholdArray = values[THRESHOLD_FIELD].split("/").map(n => parseInt(n, 10));
+        const threshold = {
+          numerator: thresholdArray[0],
+          denominator: thresholdArray[1],
+        };
+
+        return {
+          ...commonOptions,
+          type: ProposalType.AmendElectionRuleThreshold,
+          targetElectionRuleId,
+          threshold,
+        };
+      }
+      case ProposalType.AmendElectionRuleQuorum: {
+        const targetElectionRuleId = parseInt(
+          values[COMMITTEE_QUORUM_FIELD].substring(0, values[COMMITTEE_QUORUM_FIELD].indexOf(":")),
+          10,
+        );
+        const quorumArray = values[QUORUM_FIELD] && values[QUORUM_FIELD].split("/").map(n => parseInt(n, 10));
```

Comment:
Do we have a field validator that checks the user input format?

Reply:
No, I was thinking of creating some tickets to extend [medulas' field validators](https://github.com/iov-one/ponferrada/blob/master/packages/medulas-react-components/src/utils/forms/validators/index.tsx) with new validators for the create proposal form and apply them in Governance